content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -141,15 +194,15 @@ docker run \--detach -v "/DockerDir:/data" -p 9004:80 -p 9005:443 ixsystems/true
 {{< hint info >}}
 Use [Windows compatible syntax](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats) when specifying paths in the Windows file system.
 For example, if the created directory for the TrueCommand image was created in the Windows <file>Documents</file> folder, the `docker` command would be:
-`docker run \--detach -v C:\Users\\Example\\Documents\\DockerDir`.
-In this command *C* is the drive letter, *Example* is the current user name, and *DockerDir* is the TrueCommand image directory.
+`docker run \--detach -v {C}:\Users\\{Example}\\Documents\\{DockerDir}`.
+`{C}` is the drive letter, `{Example}` is the current user name, and `{DockerDir}` is the TrueCommand image directory.
 {{< /hint >}}
 
-Although there are different ways to run a Docker container, `-v /*hostdirectory*:/data` is required for TrueCommand to function.
+Although there are different ways to run a Docker container, `-v /{hostdirectory}:/data` is required for TrueCommand to function.
 
 {{< hint warning >}}
-Do not try to use the same `hostdirectory` for two different containers!
-This results in file conflicts and database corruption.
+Do not try to use the same `{hostdirectory}` for two different containers!
```

Comment:
in this instance you don't need the CLI command emphasis or the { }, you can just say host directory or say
"Do not try to use the same host directory in the command strings above for two different containers! in this case you are talking about the variable in a non-command instance so sentence formatting is correct.